src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -211,8 +211,8 @@ public double getDefaultAlleleFrequency() {
     public Mutect3DatasetMode mutect3DatasetMode = Mutect3DatasetMode.ILLUMINA;
 
     public enum Mutect3DatasetMode {
-        ILLUMINA(11),
-        ULTIMA(11);
+        ILLUMINA(11 + 20),
```

Comment:
Magic constant.  Can't this be calculated?

Reply:
done